src/server/player_sao.h

```diff
@@ -104,6 +104,8 @@ class PlayerSAO : public UnitSAO
 	f32 getFov() const { return m_fov; }
 	void setWantedRange(const s16 range);
 	s16 getWantedRange() const { return m_wanted_range; }
+	void setCameraInverted(bool camera_inverted) { m_camera_inverted = camera_inverted; } // not stored
```

Comment:
The comment is kinda confusing.
Does this mean that `m_camera_inverted` is not persistently stored in the player db? If so, better move the comment to the member var (or remove it).

Reply:
Agreed. I'll add it there.